### PR TITLE
Make static station information actually work

### PR DESCRIPTION
**Pull request by fahhem**
Without this, if you do --disable_discovery, you're never going to get anything on the dashboard because we never actually attempt to watch or listen to the stations configured via the config (in `conf.stations`).

### CODE REVIEW

File: openhtf/output/web_gui/__init__.py [hunk 1]
```diff
@@ -223,26 +223,33 @@ def __getitem__(self, hostport):  # pylint:disable=invalid-name
...
+    self._handle_stations(self.stations.iterkeys())
     while not self._stop_event.is_set():
-      self._discover()
-      self._stop_event.wait(self._discovery_interval_s)
+      if self._disable_discovery:
+        _LOG.debug('Station discovery is disabled; using static station info')
+        self._handle_stations(self.stations.iterkeys())
+      else:
+        self._discover()
+        self._stop_event.wait(self._discovery_interval_s)
```
Comment (Kenadia) [inline openhtf/output/web_gui/__init__.py]: It seems like we should wait on `_discovery_interval_s` regardless of whether `disable_discovery` is `True`.

File: openhtf/output/web_gui/__init__.py [hunk 2]
```diff
@@ -223,26 +223,33 @@ def __getitem__(self, hostport):  # pylint:disable=invalid-name
...
-    if self._disable_discovery:
-      _LOG.debug("Station discovery is disabled; StationStore won't update.")
-      return
+    self._handle_stations(self.stations.iterkeys())
     while not self._stop_event.is_set():
-      self._discover()
-      self._stop_event.wait(self._discovery_interval_s)
+      if self._disable_discovery:
+        _LOG.debug('Station discovery is disabled; using static station info')
+        self._handle_stations(self.stations.iterkeys())
```
Comment (Kenadia) [inline openhtf/output/web_gui/__init__.py]: Seems like it's reasonable for `_disable_discovery` to be `True` and `conf.stations` to be set as well, in which case, we want to handle the stations in `self.stations` even when `_disable_discovery` is False.

If that sounds right, then let's remove the call to `_handle_stations` from `_discover`. Then in this loop, we can call `_discover`, followed by a call to `self._handle_stations(self.stations.iterkeys())` to get all the stations.

File: openhtf/output/web_gui/__init__.py (hunk 1 above)
Comment (fahhem) [inline openhtf/output/web_gui/__init__.py]: Good point!

File: openhtf/output/web_gui/__init__.py (hunk 2 above)
Comment (fahhem) [inline openhtf/output/web_gui/__init__.py]: Yeah, that cleaned up a bunch, thanks!

**Pull request merged**